src/adagrams.js

```diff
@@ -1,8 +1,90 @@
 const Adagrams = {
+  // Wave 1
   drawLetters() {
-    // Implement this method for wave 1
+    const letterPool = ('A'.repeat(10) + 'B'.repeat(2) + 'C'.repeat(2) + 'D'.repeat(4) + 'E'.repeat(12) + 'F'.repeat(2) + 'G'.repeat(3) + 'H'.repeat(2) + 'I'.repeat(9) + 'J' + 'K' + 'L'.repeat(4) + 'M'.repeat(2) + 'N'.repeat(6) + 'O'.repeat(8) + 'P'.repeat(2) + 'Q' + 'R'.repeat(6) + 'S'.repeat(4) + 'T'.repeat(6) + 'U'.repeat(4) + 'V'.repeat(2) + 'W'.repeat(2) + 'X' + 'Y'.repeat(2) + 'Z').split('');
+
+    const letterHandIndices = [];
+    do {
+      let newIndex = Math.floor(Math.random() * letterPool.length);
+      if (!letterHandIndices.includes(newIndex)) {
+      letterHandIndices.push(newIndex) }
+    }
+    while (letterHandIndices.length < 10);
```

Comment:
`do while` loops aren't super popular in most js workplaces. You could rewrite this as a vanilla while loop like so:
```suggestion
    while (letterHandIndices.length < 10){
      let newIndex = Math.floor(Math.random() * letterPool.length);
      if (!letterHandIndices.includes(newIndex)) {
      letterHandIndices.push(newIndex) }
    }
```